core/src/main/java/org/mskcc/cbio/portal/util/ProgressMonitor.java

```diff
@@ -58,7 +58,7 @@
     /**
      * Private ctor for enforcing singleton
      */
-    private ProgressMonitor() {
+    public ProgressMonitor() {
```

Comment:
I don't think this should be public. ImportGDD.importData should not need an instance of ProgressMonitor passed to it. If you look in your ImportGDD main method, you are using ProgressMonitor correctly:

`ProgressMonitor.setCurrentMessage("No GDD data found in directory, skipping import: " + dataFile.getCanonicalPath());`

Try doing this throughout the importData method as well.